src/app/containers/create-movie/index.jsx

```diff
@@ -1,28 +1,22 @@
 'use client'
 
-import { useMutation } from "@tanstack/react-query"
-import axios from "axios"
-
-const createMovie = async () => {
-    const response = await axios.post('/api/movies', {id: 7, name: "Dune" })
-
-    return response.data
-}
+import CreateMovieForm from "./createMovieForm"
+import { useCreateMovie } from "../hooks/useCreateMovie";
+import DisplayError from "./displayError";
+import DisplaySuccess from "./displaySuccess";
 
 export const CreateMovie = () => {
-    const { mutate, data, isSuccess, isError, error } = useMutation({
-        mutationFn: createMovie, 
-        mutationKey: ["create-movie"]
-    })
-
-    return <>
-        {isError && <div className="text-red-600">{error.message}</div>}
-        <button onClick={() => mutate()}>Create movie</button>
-        {isSuccess && <div>
-            Newly created movie
-            <p>This is id: {data.id}</p>
-            <p>This is movie name: {data.name}</p>
-            </div>}
-    </>
+    const { mutate, data, isSuccess, isError, error } = useCreateMovie();
 
-}
\ No newline at end of file
+    const handleFormSubmit = (values) => {
+      mutate(values);
+    };
+      
+    return (
+        <div className="flex flex-col items-center justify-center h-full w-full space-y-4">
+          {isError && <DisplayError error={error} />}
+          <CreateMovieForm onSubmit={handleFormSubmit} />
+          {isSuccess && <DisplaySuccess data={data} />}
+        </div>
+    );
+};
```

Comment:
Let's leave an empty line at the end of the file

Reply:
solved at [6c63366](https://github.com/OctaFloare/flitnext-2024/pull/12/commits/6c63366ce437096d199fa5b5aafd3880fbd91fc9)